force-app/main/default/lwc/barcodeScannerExample/barcodeScannerExample.html

```diff
@@ -0,0 +1,53 @@
+<!-- barcodeScannerExample.html -->
```

Comment:
Please rename the component (and related files) so that it doesn't include "example".

Reply:
Removed `Example`! Wondering if it should be `QRCodeScanner` if we are scanning only QR codes specifically?

Reply:
Or is it okay as BarcodeScanner?